starter/js/atm.js

```diff
@@ -1,15 +1,53 @@
 //Begin with the document ready function
+$( document ).ready(function() {
+});
+
+
+// START BELOW
+
+
+
+    var checkingBalance = 0;
+    var savingsBalance = 0;
 
     //Checking account deposit function
+    var depositChecking = function() {
+      var amount = parseInt(document.querySelector('#amountChecking').value);
+      document.querySelector('#amountChecking').value = "";
+
```

Comment:
Interesting decision to go with vanilla JS and no jQuery! Was this an intentional choice?